src/server/internalServer.cpp

```diff
@@ -167,6 +172,21 @@ ParameterizedMessage nonParameterizedMessage(const std::string& msgId)
 
 } // unnamed namespace
 
+struct Error : public std::runtime_error {
+  Error(ParameterizedMessage message)
+    : std::runtime_error("Invalid_argument"),
+      _message(message)
+  {}
```

Comment:
1. Make `explicit`
2. Pass parameter by const reference
3. Fix the message passed to `std::runtime_error`